bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -177,6 +177,28 @@ def pypowarray(numpyvec, pow):
 from ._pyz_utils import MethodTemplateGetter, MethodTemplateWrapper
 
 
+# For refernces to keep alive the NumPy arrays that are read by
+# MakeNumpyDataFrame.
+_numpy_data = {}
```

Comment:
I guess it's alright to have a global dict, we have a different address for each dictionary of numpy arrays per each process, so different RDataFrame can be created for different numpy sources. Just adding the comment to double check with you

Reply:
Yes exactly, that its the idea!